src/components/camera/camera.tsx

```diff
@@ -17,7 +17,7 @@ export class CameraPWA {
 
   @Prop({ context: 'isServer' }) private isServer: boolean;
 
-  @Prop() facingMode: string = 'user';
```

Comment:
The default shoud be environment. not sure why @capacitor/core is setting the correct one BTW
![image](https://user-images.githubusercontent.com/11022437/99710142-f20b3a00-2a7e-11eb-9ea3-8149977e2592.png)

Reply:
![image](https://user-images.githubusercontent.com/11022437/99710227-0cddae80-2a7f-11eb-9722-0c939bd89a6c.png)